### PR TITLE
HTTP Requests collection to test Composition Gateway demo

### PR DESCRIPTION
**Pull request by mauroservienti**

### CODE REVIEW

File: demos/asp-net-core/postman-collection.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+				"url": "http://localhost:4457/orders",
+				"method": "GET",
+				"header": [],
+				"body": {},
+				"description": "gets a list of orders composed with information coming from different endpoits"
+			},
+			"response": []
+		}
+	]
+}
```
Comment (adamralph) [inline demos/asp-net-core/postman-collection.json]: This file currently contains tabs. Are we OK with that, or would we prefer spaces? (I'd prefer spaces.)

File: demos/asp-net-core/postman-collection.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	"variables": [],
+	"info": {
+		"name": "ASP.NET Core demo - CompositionGateway",
+		"_postman_id": "71510306-c9e1-ad86-ca26-d98bd6fbfad2",
+		"description": "",
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Order #1, SalesOnly",
```
Comment (adamralph) [inline demos/asp-net-core/postman-collection.json]: Is this designed to be a human friendly name? If so, should it be `"Order #1, Sales only"`?

Comment (mauroservienti) [reply demos/asp-net-core/postman-collection.json]: Good point I'll fix it

File: demos/asp-net-core/postman-collection.json (hunk 1 above)
Comment (mauroservienti) [reply demos/asp-net-core/postman-collection.json]: It's the way Postman exports it, is it worth to change it to spaces? It's not meant to be used by a  human or manually edited.

Comment (adamralph) [reply demos/asp-net-core/postman-collection.json]: Ah, in that case let's leave it as tabs.

**Comment by mauroservienti** [comment]
@adamralph your changes look good, I pushed some changes to rename Postman requests. If you're happy, please, merge.

**Comment by mauroservienti** [comment]
squashed, ready to be merged @adamralph

**Pull request merged**